app/javascript/components/forms/index.js

```diff
@@ -1,22 +1,26 @@
 import AuthTokenInput   from './AuthTokenInput'
 import CategoryFields   from './CategoryFields'
+import CommentsField    from './CommentsField'
 import ContactFields    from './ContactFields'
 import DeleteButton     from './DeleteButton'
 import ErrorMessages    from './ErrorMessages'
 import LocationFields   from './LocationFields'
 import NameField        from './NameField'
 import ServiceAreaField from './ServiceAreaField'
 import SpacerField      from './SpacerField'
+import SubmitButton     from './SubmitButton'
 
 export {
   AuthTokenInput,
   CategoryFields,
+  CommentsField,
```

Comment:
was comments not on offer form?

Reply:
It was, i just hadn't extracted it.

BTW, this is currently saving to the `description` field, which it probably shouldn't be.

Reply:
yeah, `description` is for the metadata accordion for each checkbox

i'm adding in a `SubmissionResponse` model to be able to import prior data that doesn't match our forms, so this comment could later get stored as one of those